tests/test_color.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+
+from jrnl.color import colorize
+from colorama import Fore, Style
+
+
+@pytest.fixture()
+def data_fixture():
+    string = "Zwei peanuts walked into a bar"
```

Comment:
Just curious, what's this a reference to?

Reply:
Hah, I tried to quote a "translation" of [The Funniest Joke in the World](https://en.wikipedia.org/wiki/The_Funniest_Joke_in_the_World) from memory, but I completely butchered it. I can change it the actual quote, or replace it with some more being gibberish